OS-SPEC.md

```diff
@@ -7,12 +7,23 @@ This document attempts to define a minimal set of things that must exist for mos
 
 ACIs that define the label `os=linux` can expect this environment by default.
 
+## Execution Environment
+
+It is RECOMMENDED that the following environment variables be set for each application's main process and any lifecycle processes:
```

Comment:
Nothing says an implementation CAN'T do it, I just don't know that this
adds much to the spec.

On Thu, May 7, 2015 at 4:13 PM, Jonathan Boulle <notifications@github.com>
wrote:

> In OS-SPEC.md <https://github.com/appc/spec/pull/273#discussion_r29905247>
> :
>
> > @@ -7,12 +7,23 @@ This document attempts to define a minimal set of things that must exist for mos
> >
> >  ACIs that define the label `os=linux` can expect this environment by default.
> >
> > +## Execution Environment
> > +
> > +It is RECOMMENDED that the following environment variables be set for each application's main process and any lifecycle processes:
>
> I was anticipating that there is still a nontrivial amount of software
> that does depend on these things and so it would be worth gently nudging
> implementers in that direction. But I don't feel strongly enough about it
> to keep fighting this :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/spec/pull/273/files#r29905247>.
>